examples/meam/msmeam/library.MSmeam

```diff
@@ -0,0 +1,14 @@
+# DATE: 2018-09-22  MB DATE: 2016-09-21 CONTRIBUTOR: Steve Valone, smv@lanl.gov CITATION: Baskes, PRB 1992; smv, sr, mib, JNM 2010
```

Comment:
There should be a UNITS: tag here and the file should be moved to `potentials/library.msmeam` (note the case).